From jrm8005@gmail.com  Thu Dec  4 11:40:05 2008
From: "J.R. Mauro" <jrm8005@gmail.com>
Date: Fri, 14 Nov 2008 20:01:14 -0500
Subject: Staging: comedi: fix build if CONFIG_PROC_FS is not set
To: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Greg KH <greg@kroah.com>, Stephen Rothwell <sfr@canb.auug.org.au>, abbotti@mev.co.uk, fmhess@users.sourceforge.net, ds@schleef.org
Message-ID: <20081115010114.GA29057@tuxbook-pro>
Content-Disposition: inline


Don't include procfs functions if CONFIG_PROC_FS is not set

Signed-off by: J.R. Mauro <jrm8005@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/proc.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/staging/comedi/proc.c
+++ b/drivers/staging/comedi/proc.c
@@ -85,6 +85,7 @@ int comedi_read_procmem(char *buf, char 
 	return l;
 }
 
+#ifdef CONFIG_PROC_FS
 void comedi_proc_init(void)
 {
 	struct proc_dir_entry *comedi_proc;
@@ -98,3 +99,4 @@ void comedi_proc_cleanup(void)
 {
 	remove_proc_entry("comedi", 0);
 }
+#endif
